shell/src/main/java/tachyon/shell/command/FileInfoCommand.java

```diff
@@ -54,7 +54,8 @@ void runCommand(TachyonURI path) throws IOException {
       throw new IOException(path + " is a directory path so does not have file blocks.");
     }
 
-    System.out.println(path + " with file id " + fd.getFileId() + " has the following blocks: ");
+    System.out.println(fInfo);
```

Comment:
From a code reviewing and code reading perspective it's difficult to tell whether these two prints make sense together. Can we keep the output the same for this PR and change it in a separate PR?